content/contributing/tutorial-guidelines.md

```diff
@@ -0,0 +1,85 @@
+5. Adults want their training to be problem focused.
+6. Adults learn best when motivation comes intrinsically.
+
+Add into that Ralph Smedley’s quote, “We learn best in moments of enjoyment.” Taking a lighter touch helps to relax the learner so that the material flows into their brain with less resistance.
+
+
+# Sample Code vs. Tasks vs. Concepts vs. Tutorials
+
+To date, there have been some blurred lines where different types of documentation show up as _Tutorials._ Here are some comparisons that help define the distinction.
+
+
+## Sample Code
+
+Sample code is well commented snippets or applications that illustrate best practices for implementing a feature of the API. Sample code is modular and reusable with little customization required. 
+
+Sample code is desirable, because advanced users can typically scan the example and use it immediately without a formal tutorial. It can also be used by others as a basis for tutorials. Sample code developers can focus on what they do well, while technical writers and support personnel can use the samples to create quality training materials.
+
+
+## Tasks
+
+Tasks are step-by-step instructions for how to accomplish a specific result. For example, “Installing rippled on a Red Hat Linux Server.” Task documentation is not intended to be particularly educational. It frequently describes tasks that are only performed one time per implementation, or maintenance tasks that always follow a familiar pattern. Tasks provide troubleshooting guidance, since there are likely variables that the user must adjust based on the specifics of their use case.
+
+
+## Concepts
+
+Conceptual information describes elements of the API, how they work, and when to use them. If a tutorial requires lengthy explanations before or during the programming tasks, consider how you might separate the exposition into a new topic, or link to existing topics that set the proper context. 
+
+For example, three paragraphs of context and a single line of code would be a concept, not a tutorial.
+
+
+## Tutorials
+
+Tutorials begin with sample code that illustrates best practices for implementing a feature. They take the developer step-by-step through the development process, explaining the purpose of each block of code.
+
+Tutorials further combine a number of features to work together to solve a business problem. They describe the straightforward sunny day path to complete a task. Then, the tutorial might suggest modifications that let the developer try several different scenarios. Due to their focus on a certain limited scope of behavior, tutorials should not require extensive troubleshooting information.
+
+
+## Use Cases
+
+Use cases describe how to pull together features of the XRPL to create a practical application that solves a business problem. They provide context and assistance with the decision making process for what elements to include, then provide links to the appropriate topics for each step of implementation. 
```

Comment:
I spotted an opportunity to make this paragraph even more concise. (Totally optional)

```suggestion
Use cases describe how to pull together multiple features to create a practical application that solves a business problem. They provide context and assistance with the decision making process, then provide links to the appropriate topics for each step of implementation. 
```